website/server/controllers/api-v3/user.js

```diff
@@ -28,7 +28,29 @@ let api = {};
  * @apiGroup User
  *
  * @apiSuccess {Object} data The user object
- */
+ *
+ * @apiDescription The user profile contains data related to the authenticated user including (but not limited to);
+ * Achievements
+ * Authentications (including types and timestamps)
```

Comment:
The example was so long that I thought the description would be better. Would an annotated example be acceptable?

It'll still be a hundred lines or so...

'
{
  "success": true,
  "data": {
    "_id": "11be077e-4a54-4d3a-96b6-8f14c101aeff",
    "migration": "20170131_habit_birthday.js",
    "loginIncentives": 9,
    "webhooks": [],
    "pushDevices": [],
    "extra": {},
    "tasksOrder": {
      "rewards": [--TASKS LISTED BY ID],
      "todos": [--TASKS LISTED BY ID],
      "dailys": [--TASKS LISTED BY ID],
      "habits": [--TASKS LISTED BY ID],
    },
    "inbox": {
      "optOut": false,
      "messages": {},
      "blocks": [],
      "newMessages": 0
    },
    "tags": [
      {
        "name": "Work",
        "id": "94d3a62b-2f69-4b75-b930-d93dd2c21038"
      }
    ],
    "notifications": [    ],
    "stats": {
        --stats, training, buffs--    },
    "profile": {
      "name": "Ezra"
    },
;